### PR TITLE
Issue 3602

### PR DESCRIPTION
**Pull request by rmroot**
connects #3602 

All report charts using plotly. 
Updated printing styling for all rollup reports
Deleted unneeded components after creating shared components for similar tables and charts
Restores printing capabilities to rollup report

### CODE REVIEW

File: src/app/report-rollup/phast-rollup/phast-rollup-print/phast-rollup-print.component.html [hunk 1]
```diff
@@ -9,7 +92,7 @@ <h3>Process Heating Rollup Report</h3>
   <!-- <div class="row justify-content-center print-avoid-break" *ngFor="let calculator of calculators">
     <app-pre-assessment-print [calculator]="calculator" [settings]="settings">
     </app-pre-assessment-print>
```
Comment (nbintertech) [inline src/app/report-rollup/phast-rollup/phast-rollup-print/phast-rollup-print.component.html:94]: Commented out code

File: src/app/report-rollup/rollup-summary-energy-table/rollup-summary-energy-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  energyUnit: string;
+  @Input()
+  equipmentType: string;
+
+  totalEnergyUse: number;
+  totalCost: number;
+  constructor() { }
+
+  ngOnInit(): void {
+    console.log(this.pieChartData);
```
Comment (nbintertech) [inline src/app/report-rollup/rollup-summary-energy-table/rollup-summary-energy-table.component.ts:23]: console.log

File: src/app/phast/sankey/sankey.component.ts [hunk 3]
```diff
@@ -90,6 +90,9 @@ export class SankeyComponent implements OnInit {
       this.location = this.location.replace(/[\])}[{(]/g, '');
       this.location = this.location.replace(/#/g, "");
     }
+    console.log(this.modIndex);
+    console.log(this.location);
```
Comment (nbintertech) [inline src/app/phast/sankey/sankey.component.ts:94]: A number of console.logs in various files. Also some stray comments around but they seem useful/explanatory.

**Comment by rmroot** [comment]
CI not working. Closing and trying again

**Pull request closed without merging**